### PR TITLE
edited condor datacard making command to remove do-nuisance flag

### PR DESCRIPTION
**Pull request by abasnet97**
This PR fixes the issue with the `--do-nuisance` flag not working properly when running datacard maker (make_cards.py) using condor. The issue was that even when the flag was not explicitly called, the resulting datacards would still have the systematics included. It turned out that the datacard making command passed to condor had `--do-nuisance` flag always included. Getting rid of that and adding a new option to use the flag in `run_condor` method resolved the issue.

### CODE REVIEW

**Comment by kmohrman** [comment]
Thanks very much for fixing this! Just wanted to check that you've verified that these changes did not inadvertently cause any changes when running _without_ condor (i.e. explicitly verified that including or excluding the `--do-nuisance` flag now has the expected effect both without condor and with condor?)

**Comment by abasnet97** [comment]
Yes. I have double-checked that. I ran the script using condor without passing the flag, and I see that the resulting datacard has no systematics included. Again, ran the script using condor with the flag included, and the resulting datcard has systematics included.

File: analysis/topEFT/make_cards.py [hunk 1]
```diff
@@ -197,12 +199,12 @@ def main():
     if isinstance(wcs,str):
         wcs = wcs.split(",")
 
-    if use_condor:
+    #if use_condor:
```
Comment (bryates) [inline analysis/topEFT/make_cards.py:202]: You can probably just remove this now

File: analysis/topEFT/make_cards.py [hunk 2]
```diff
@@ -197,12 +199,12 @@ def main():
...
         # Note:
         #   The dc in the parent submission is only used to generate the selectedWCs file and figure
         #   out what channels/samples are available, so we can just drop the systematics to speed
         #   things up
-        do_nuis = False
+        #do_nuis = False
```
Comment (bryates) [inline analysis/topEFT/make_cards.py:207]: And this as well

**Review by bryates — APPROVED** [review]
Looks good. Just 2 minor comments on commented out code.

File: analysis/topEFT/make_cards.py (hunk 1 above)
Comment (abasnet97) [reply analysis/topEFT/make_cards.py]: Yes, I forgot about that. Done now!

File: analysis/topEFT/make_cards.py (hunk 2 above)
Comment (abasnet97) [reply analysis/topEFT/make_cards.py]: Done as well.

**Comment by kmohrman** [comment]
Thanks again for this PR Aashwin. Since it looks like all of the comments have been addressed, I'd be happy to go ahead and merge it now.

**Pull request merged**